datafusion/Cargo.toml

```diff
@@ -77,6 +77,7 @@ rand = "0.8"
 avro-rs = { version = "0.13", features = ["snappy"], optional = true }
 num-traits = { version = "0.2", optional = true }
 pyo3 = { version = "0.14", optional = true }
+sqllogictest = "0.1.0"
```

Comment:
i think this should go into dev-dependencies instead?

Reply:
yep